guides/source/action_mailer_basics.md

```diff
@@ -777,7 +777,39 @@ class UserMailer < ApplicationMailer
 end
 ```
 
-* You could use an `after_delivery` to record the delivery of the message.
+* You could use an `after_deliver` to record the delivery of the message. It
```

Comment:
This rename was just catching a typo. The callback name changed from `_delivery` to `_deliver` in the original PR.